docs/user/02-10-usage.md

```diff
@@ -1,28 +1,33 @@
 # Use BTP Manager to Manage SAP BTP Service Operator 
 
-## Create and Install Secret
+## Create and Install a Secret
 
 To create a real BTP Manager Secret, follow these steps:
-1. Clone the `btp-manager` repository to your local file system.
-2. Create ServiceBinding to obtain the access credentials to the ServiceInstance as described in points 2 of the [Setup](https://github.com/SAP/sap-btp-service-operator#setup) section in the SAP BTP service operator documentation.
-3. Copy and save the access credentials into your `hack/creds.json` file in the cloned `btp-manager` repository.
-4. Call [`create-secret-file.sh`](https://github.com/kyma-project/btp-manager/blob/main/hack/create-secret-file.sh). 
+1. Create ServiceBinding to obtain the access credentials to the ServiceInstance as described in point 2 of the [Setup](https://github.com/SAP/sap-btp-service-operator#setup) section in the SAP BTP service operator documentation.
```

Comment:
```suggestion
1. Create a ServiceBinding to obtain the access credentials to the ServiceInstance. For details, see [BTP Service Operator Setup: Obtain the access credentials](https://github.com/SAP/sap-btp-service-operator#setup).
```
We're in hot water if they ever change the number/order of their steps in their document. I'd rather mention the title of the step in some way instead of the number.

Reply:
Good point, thank you :)